lib/msf/base/serializer/readable_text.rb

```diff
@@ -591,16 +591,21 @@ def self.dump_advanced_options(mod, indent = '')
       'Indent'  => indent.length,
       'Columns' =>
         [
-          'Name',
-          'Current Setting',
-          'Required',
-          'Description'
+          'Name,',
+          'Current Setting,',
+          'Required,',
+          'Description,.'
```

Comment:
A suggestion, table additions for punctuation may be reasonable to implement as an option passed to the `Rex::Text::Table` base on a if `screen reading` is enabled.

Reply:
Indeed, exactly where I was thinking made the most sense to put this capability.